.github/workflows/check-pull-request.yml

```diff
@@ -27,7 +27,6 @@ env:
 
 jobs:
   fmt:
-    if: ${{ !startsWith(github.head_ref, 'release/') }}
```

Comment:
Here, you might disagree with me, but I thought as releases also have changes in the rust file, we should definitely run all tests again. What are your thoughts here @DidacSF?

Reply:
Yes, I think it's a good idea!